### ISSUE TITLE
Document core autocons

### ISSUE THREAD

**Issue by courajs** [urbit/docs#465]
Found in the wild: https://github.com/Fang-/djay/blob/261766af0383eaa6f1e168b6c4c06da69a64175e/app/djay.hoon
> ~palfun-foslup> yeah, it gets treated as equivalent to =>  |% ... --  |_ ... --

Not mentioned in the [bar rune reference](  https://urbit.org/docs/reference/hoon-expressions/rune/bar/) or the [irregular form reference](https://urbit.org/docs/reference/hoon-expressions/irregular/).

**Comment by ohAitch** [comment]
Nor the ford reference? This isn't specific to bar runes

**Comment by rmariani** [comment]
Thanks for catching this, I will look into it.

**Comment by baudtack** [comment]
@Fang- Is this only a thing in gall apps?

**Comment by Fang-** [comment]
Not sure, but would be surprised if it's special-cased like that. Probably just a compiler thing?

**Comment by ohAitch** [comment]
Yes, the ford parser effectively wraps the whole file in a `=~ ... ==`

**Comment by belisarius222** [comment]
@ohAitch is correct.  This is behavior specific to the Ford parser, not a compiler feature.  If you're writing a file that Ford does not compile, such as a vane, you have to have a single Hoon expression instead of a list of them.
